projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -38,7 +38,6 @@ aca-search-results {
 
         /* stylelint-disable-next-line */
         .mdc-evolution-chip__text-label {
```

Comment:
I guess we should probably update this selector with $mat-evolution-chip-text-label from mat-selectors.scss

Reply:
yes, I'm not touching it with this PR as this code was already there, but we should get rid of this selector entirely